### PR TITLE
Add a command to download artifacts

### LINKED ISSUE #136: Add a command to download an artifact
It would be helpful to add a command to download an artifact from a determined link. This command will then be used from the cli

### PR DESCRIPTION
**Pull request by Luni-4**
This PR fixes #136 

Thanks in advance for your review! :)

### CODE REVIEW

File: taskboot/utils.py [hunk 1]
```diff
@@ -109,15 +110,20 @@ def download_artifact(queue, task_id, artifact_name):
...
-    # Download the artifact in a temporary file
-    _, ext = os.path.splitext(artifact_name)
-    _, path = tempfile.mkstemp(suffix="-taskboot{}".format(ext))
+    if output_directory is None:
+        # Download the artifact in a temporary file
+        _, ext = os.path.splitext(artifact_name)
+        _, path = tempfile.mkstemp(suffix="-taskboot{}".format(ext))
+    else:
+        # Download the artifact in a specific directory
+        path = output_directory + artifact_name
```
Comment (marco-c) [inline taskboot/utils.py:119]: Use `os.path.join` instead of string concatenation

**Comment by Luni-4** [comment]
What's the error in the CI?

File: taskboot/cli.py [hunk 2]
```diff
@@ -251,7 +277,7 @@ def main():
         help="Github repository name to use (example: mozilla/task-boot)",
     )
     github_release_cmd.add_argument(
-        "version", type=str, help="Release version tag to create or update on github"
+        "version", type=str, help="Release version tag to create or update on github",
```
Comment (marco-c) [inline taskboot/cli.py:280]: I imagine the formatting changes were applied by black and not you, right?

Comment (Luni-4) [reply taskboot/cli.py:280]: Exactly

File: taskboot/utils.py (hunk 1 above)
Comment (Luni-4) [reply taskboot/utils.py:119]: `output_directory` is a `pathlib.Path`, perhaps it would be better to use the `/` operator. What do you think?

Comment (marco-c) [reply taskboot/utils.py]: WFM

**Comment by marco-c** [comment]
> What's the error in the CI?

The Docker build failed, let's see if it still fails, maybe it was an intermittent failure.

**Comment by Luni-4** [comment]
Nope, the Docker build task fails again :(

**Comment by marco-c** [comment]
Yep, not sure what's wrong, the last build from 6 days ago passed.

**Pull request merged**